test/SymmetricPluginManagerTest.php

```diff
@@ -44,6 +45,9 @@ public function testHas($symmetric)
     public function testGet($symmetric)
     {
         $plugin = new SymmetricPluginManager();
+        if (! extension_loaded($symmetric)) {
```

Comment:
I think we can skip the instantiation step if extension not loaded, and set expected exception, because in next line, the extension used in the tests.

Warm regards,

Abdul Malik Ikhsan

Pada 6 Mei 2016, pukul 15.44, Enrico Zimuel <notifications@github.com> menulis:

> In test/SymmetricPluginManagerTest.php:
> 
> > @@ -44,6 +45,9 @@ public function testHas($symmetric)
> >      public function testGet($symmetric)
> >      {
> >          $plugin = new SymmetricPluginManager();
> > +        if (! extension_loaded($symmetric)) {
> @samsonasik why? The symmetric plugins are loaded only on the get() execution, not in the constructor of SymmetricPluginManager.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>